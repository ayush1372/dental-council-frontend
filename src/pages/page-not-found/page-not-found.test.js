import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';

import { PageNotFound } from './index';

test('PageNotFound  snapshot', () => {
  const { container } = render(
    <Router>
      <PageNotFound />
    </Router>
  );
  expect(container).toMatchSnapshot();
});
