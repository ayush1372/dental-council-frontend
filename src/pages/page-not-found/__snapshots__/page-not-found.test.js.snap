// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound  snapshot 1`] = `
<div>
  <div
    class="main"
  >
    <header>
      <h1>
        Page not found (404)
      </h1>
    </header>
  </div>
</div>
`;
